### ISSUE TITLE
Pretrained model?

### ISSUE THREAD

**Issue by comex** [f90/AdversarialAudioSeparation#2]
Hi,

I know this is a bit late, it would be nice if a pretrained model were available for download, to easily recreate the original results and for use on custom audio.

In lieu of that, I'm trying to recreate the experiment, but I'm having some difficulty.  Although the readme helpfully explains what to do, I'm not sure if I can obtain the same datasets.  iKala is apparently no longer available at all, and MedleyDB is only available on request.  I guess I'll try training using only the other two...

**Comment by f90** [comment]
Hey,

you are right, it appears the iKala dataset is no longer available, or at least no new licenses are being given out anymore to be able to use it. I would expect to still see an improvement using our semi-supervised training method, since the dataset partitions are still set up in such a way so that the unsupervised dataset sources are also included in the test set, but not in the supervised training set and so provide important additional information to the separator.

Unfortunately I don't have the pre-trained models anymore... I might come around and re-train them locally and put them on-line.
